horovod/common/controller.cc

```diff
@@ -130,14 +127,17 @@ ResponseList Controller::ComputeResponseList() {
   }
 
   cache_coordinator.set_should_shut_down(should_shut_down);
+  if(should_shut_down){
+    LOG(DEBUG) << "Have stalled tensors, will shut down.";
```

Comment:
This is not necessarily true, since shut_down could have been set from outside.

Reply:
Debugging stuff. Removed.